langchain/src/vectorstores/vectara.ts

```diff
@@ -163,6 +168,62 @@ export class VectaraStore extends VectorStore {
     }
   }
 
+  async addFiles(
+    file_paths: string[],
+    metadata: Record<string, unknown> | undefined = undefined
+  ) {
+    const docIds: string[] = [];
+    // TODO: What do I set doc ids to?
```

Comment:
With the FILE_UPLOAD API, the doc_id is automatically set for Vectara based on the file name.